app/kuma-dp/cmd/run.go

```diff
@@ -92,48 +93,27 @@ func newRunCmd() *cobra.Command {
 				runLog.Info("generated Envoy configuration will be stored in a temporary directory", "dir", tmpDir)
 			}
 
-			runLog.Info("starting Dataplane (Envoy) ...")
-
 			dataplane := envoy.New(envoy.Opts{
 				Catalog:   catalog,
 				Config:    cfg,
 				Generator: bootstrapGenerator,
 				Stdout:    cmd.OutOrStdout(),
 				Stderr:    cmd.OutOrStderr(),
 			})
+			server := accesslogs.NewAccessLogServer(cfg.Dataplane)
 
-			server := accesslogs.NewAccessLogServer()
-			defer server.Close()
-
-			logServerErr := make(chan error)
-			go func() {
-				defer close(logServerErr)
-				if err := server.Start(cfg.Dataplane); err != nil {
-					runLog.Error(err, "problem running Access Log server")
```

Comment:
Why did you eliminate that log line, it looks useful for troubleshooting process

Reply:
It's now handled by: `runLog.Error(err, "error while running Kuma DP")`
If any component returns an error the `componentMgr.Start` returns an error.

Reply:
Yes, but it will return `err := s.server.Serve(lis)` which is actually a grpc error. So we won't understand which exactly component failed to start.